Rear Rider/Rear Rider/BlueTooth/BLEManager.swift

```diff
@@ -203,32 +199,42 @@ class BLEManager: NSObject, ObservableObject, CBCentralManagerDelegate, CBPeriph
     func peripheral(_ peripheral: CBPeripheral, didUpdateValueFor characteristic: CBCharacteristic, error: Error?) {
         if characteristic == reverseCharacteristic {
             let ASCIIString = NSString(data: characteristic.value ?? Data(), encoding: String.Encoding.utf8.rawValue)
-            let newMessage = Message(id: messages.count, msg: "R: \(ASCIIString! as String)")
-            messages.append(newMessage)
+            log.addLog(from: "BT", message: "Recv(reverse): \(ASCIIString! as String)")
             print("Value received \(ASCIIString! as String).")
         }
         else if characteristic == notifyCharacteristic {
             let ASCIIString = NSString(data: characteristic.value ?? Data(), encoding: String.Encoding.utf8.rawValue)
-            let newMessage = Message(id: messages.count, msg: "R: \(ASCIIString! as String)")
-            messages.append(newMessage)
+            log.addLog(from: "BT", message: "Recv(notify): \(ASCIIString! as String)")
             print("Value received \(ASCIIString! as String).")
         }
     }
     
     func peripheral(_ peripheral: CBPeripheral, didModifyServices invalidatedServices: [CBService]) {
         connected = false
-        startScanning()
+        log.addLog(from: "BT", message: "RPi disconnected")
     }
     
-    /// Appends a message to the status messages array
-    /// - Parameter msg: a String describing the message
-    private func addStatusMessage(message msg: String) {
-        let newStatus = StatusMsg(id: statusMsgs.count, msg: msg)
-        statusMsgs.append(newStatus)
-        print(msg)
+    func centralManager(_ central: CBCentralManager, didDisconnectPeripheral peripheral: CBPeripheral, error: Error?) {
```

Comment:
mind putting some doc comments above this func?

Reply:
Yes I have a bunch of functions that I need to comment.